posts/2024-06-28-gno-debugger/README.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Debugging Gno Programs"
+publication_date: 2024-06-28T13:37:00Z
+slug: gno-debugger-blog-post
+tags: [blog, post, tutorial, gno, debugger]
+authors: [@mvertes]
+---
+# Debugging Gno Programs
+ 
+In this article, we introduce the new Gno debugger feature and show how it can be used to better understand gno programs and help fixing bugs.
```

Comment:
```suggestion
In this article, we introduce the new Gno debugger feature and show how it can be used to better understand Gno programs and help fixing bugs.
```